docs/databases/uploads.md

```diff
@@ -42,6 +43,10 @@ When people upload a CSV to a collection, Metabase will:
 - Create a table to store that data in the database and schema that the Admin selected to store uploads.
 - Create a [model](../data-modeling/models.md) that wraps the uploaded table, and save that model to the collection the person uploaded the CSV data to.
 
+### Add people to a group with unrestricted data access to that database
```

Comment:
Should we be more explicit in the header and say:
unrestricted access to the upload schema?